src/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
```

Comment:
It would be better for you if you fixed your linter errors manually rather than disabling it.